sdk/typescript/introspector/test/invoke.spec.ts

```diff
@@ -245,4 +245,30 @@ describe("Invoke typescript function", function () {
       })
     }
   })
+
+  describe("Should correctly handle aliases", async function () {
+    this.timeout(60000)
+
+    // Mocking the fetch from the dagger API
+    it("Should correctly invoke hello world", async function () {
```

Comment:
Are you planning on adding more sub test cases inside this `describe`? Otherwise the `it` can be pulled out.

Reply:
I plan to add more tests later!